sklearn/feature_selection/mutual_info.py

```diff
@@ -0,0 +1,298 @@
+# Authors: Andrea Bravi <a.bravi@uottawa.ca>
+#          Nikolay Mayorov <n59_ru@hotmail.com>
+# License: 3-clause BSD
+
+from __future__ import division
+
+import numpy as np
+from scipy.sparse import issparse
+from scipy.special import digamma
+
+from ..base import BaseEstimator
+from .base import SelectorMixin
+from ..metrics.cluster.supervised import mutual_info_score
+from ..utils.validation import check_X_y
+from ..neighbors import NearestNeighbors
+from ..preprocessing import scale
+
+
+def _compute_mi_cc(x, y, n_neighbors, scale_variables):
+    """Compute mutual information between two continuous variables.
+
+    Parameter `scale_variables` determine whether to scale variables to
```

Comment:
This seems like more of an explanation to `scale_variables` than to `_compute_mi_cc` . Can you follow the template used in other functions? i.e having a `Parameters` and `Returns` section